packages/connect-explorer/src/reducers/methodReducer.ts

```diff
@@ -145,6 +150,9 @@ const setAffectedValues = (state: MethodState, field: any) => {
             const affectedField = root.find(f => f.name === af);
             if (affectedField) {
                 affectedField.value = values[index];
+                if (state.name === 'composeTransaction') {
+                    affectedField.value = values;
+                }
```

Comment:
ok, this seems like some followup. I don't understand at the moment why it is needed, but I am able to live with it now